presto-tests/src/main/java/Dummy.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+public class Dummy
```

Comment:
put this in `io.prestosql.tests` package.
does it need to be public?

Reply:
These are tests that are only run during the build and don’t need to be published at all? Or you have external consumers of these tests?

Reply:
I think it is safer to publish them. We may want to run them internally.

Reply:
If they are going to be used as a general testing facility then put the classes in src/main with a small hook in src/test to trigger them in the build. Then if you want to use them as a tool elsewhere then you can put a main method with a testng runner in src/main so that you can:

`java -jar presto-tests-X.jar --param1=x --param2=y --whatever`

Reply:
`presto-tests` are not intended to be used by other modules/projects. Just the same way as eg `presto-sqlserver` is not.
I would prefer to publish the artifact as-is (hence i approved). I am OK with skipping it from publishing.
(and I don't want to move the tests into src/main today; currently, there is no plan to make them "portable")